posts/2020-05-15-six-years-of-rust.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "Six Years of Rust"
+author: The Rust Core Team
+---
+
+Today marks [Rust]'s sixth birthday since it went 1.0 in 2015. A lot has changed since then and especially over the past year, and Rust was no different. In 2020, there was no foundation yet, no const generics, and a lot organisations were still wondering whether Rust was production ready. 
+
+In the midst of the COVID-19 pandemic, hundreds of Rust's global distributed set of team members and volunteers shipped over 9 major releases of Rust and multiple patch releases. Today, "Rust in production" isn't a question, but a statement. The newly founded Rust foundation has several members who value using Rust in production enough to help continue to support and contribute to its open development ecosystem.
+
+We wanted to take today to look back at some of the major improvements over the past year, how the community has been using in Rust in production, and finally look ahead at some of the work that is currently ongoing to improve and use Rust for small and large scale projects over the next year. Let's get started!
+
+[rust]: https://www.rust-lang.org
+
+## Recent Additions
+The Rust language has improved tremendously in the past year, gaining a lot of quality of life features, that while they don't fundamentally change the language, they help make using and maintaining Rust in more places even easier.
+
+-  As of Rust 1.52.0 and the upgrade to LLVM 12, one of few cases of unsoundness around forward progress (such as handling infinite loops) has finally been resolved. This has been a long running collaboration between the Rust teams and the LLVM project, and is a great example of improvements to Rust also benefitting the wider ecosystem of programming languages.
+
+-  On supporting an even wider ecosystem, the introduction of tier 1 support for 64 bit ARM Linux, and tier 2 support for ARM macOS & ARM Windows, has made Rust an even better place to easily build your projects across new and different architectures.
```

Comment:
Do we capitalize `Tier` here? I think we do? Not sure.

Reply:
It looked funny to me too :)